project/NoIDEExport.scala

```diff
@@ -0,0 +1,31 @@
+package build
+
+import sbt._
+import Keys._
+
+object NoIDEExport {
+  lazy val bloopGenerateKey: Option[TaskKey[Option[File]]] = {
+    val optBloopKeysClass = try {
+      Some(Class.forName("bloop.integrations.sbt.BloopKeys"))
+    } catch {
+      case _: ClassNotFoundException => None
+    }
+
+    optBloopKeysClass.map { bloopKeysClass =>
+      val bloopGenerateGetter = bloopKeysClass.getMethod("bloopGenerate")
+      bloopGenerateGetter.invoke(null).asInstanceOf[TaskKey[Option[File]]]
+    }
+  }
+
+  lazy val noIDEExportSettings: Seq[Setting[_]] = {
+    bloopGenerateKey match {

```

Comment:
I think this deserves a comment that bloop is metals (pretty much the comment in the commit message will do IMO).